crates/bevy_text/src/text.rs

```diff
@@ -249,3 +254,57 @@ impl Default for TextStyle {
         }
     }
 }
+
+/// Used to configure the default font used by [`TextStyle`] when the font is set to [`FontRef::Default`]
+#[derive(Default, Resource)]
+pub struct DefaultFont {
+    pub path: Option<AssetPath<'static>>,
+}
+
+/// Loads fonts defined by a [`FontRef`]
+/// If the font has already been loaded then it simply converts the [`FontRef::Path`] to a [`FontRef::Handle`]
+/// Otherwise it tries to load the font with the [`AssetServer`]
+pub fn load_font(
+    mut query: Query<&mut Text, Added<Text>>,
```

Comment:
This will only run for added text. FontRef should would consistently, even if changed at runtime.

Reply:
Ah, right, I need to handle situations where fonts are updated, I didn't think about that one.